actors/miner/tests/compact_sector_numbers_tests.rs

```diff
@@ -49,7 +49,7 @@ mod compact_sector_numbers_test {
             );
             expect_abort(
                 ExitCode::USR_ILLEGAL_ARGUMENT,
-                h.pre_commit_sector(&rt, precommit, util::PreCommitConfig::default(), false),
+                h.pre_commit_sector(&rt, precommit, util::PreCommitConfig::default(), false, false),
```

Comment:
We shouldn't have to pass this flag. This pattern doesn't scale to all the possible expectations and abort points that a method might hit.

The mock should always set expectations for the events. If the method aborts before emitting them, then at line 54 you should `rt.reset()` to void any remaining expectations. Most places will probably already do this, but this one forgot.

Reply:
This has been fixed in all tests. Done.